modules/nodes-nodes-resources-configuring-auto.adoc

```diff
@@ -73,7 +73,7 @@ spec:
 ----
 <1> Assign a name to CR.
 <2> Add the `autoSizingReserved` parameter set to `true` to allow {product-title} to automatically determine and allocate the `system-reserved` resources on the nodes associated with the specified label. To disable automatic allocation on those nodes, set this parameter to `false`.
-<3> Specify the label from the machine config pool.
+<3> Specify the label from the machine config pool. You can choose any desired labels for machine config pool, such as `custom-kubelet=small-pods`, or the default label, `pools.operator.machineconfiguration.openshift.io/worker`.
```

Comment:
Should there be a "the"? "for the machine config pool"?

Also, the two labels you list are formatted differently. The first one has an = with "small-pods" after it, but the second one doesn't have an =. Should it? (even though it's empty). 

Or maybe they should be shown as they would be in the YAML?

```suggestion
<3> Specify the label from the machine config pool. You can choose any desired labels for machine config pool, such as `custom-kubelet: "small-pods"`, or the default label, `pools.operator.machineconfiguration.openshift.io/worker: ""`.
```

Reply:
Thanks. Updated to match the published docs entries. 

However, I need to ask an engineer if custom-kubelet: "small-pods"` is correct as the YAML file entry.

Thanks!

Reply:
`custom-kubelet: small-pods` is fine. Quotation marks are not required when writing the `matchLabels`